code/processes/gateway.q

```diff
@@ -100,10 +100,16 @@ queryqueue:([queryid:`u#`long$()] time:`timestamp$(); clienth:`g#`int$(); query:
 // client details
 clients:([]time:`timestamp$(); clienth:`g#`int$(); user:`symbol$(); ip:`int$(); host:`symbol$())
 
+//Function to generate random placeholder
+genrand:{system ("S ",string `int$.z.T);-1?0Ng}
```

Comment:
don't need the brackets here

Reply:
cool, will take out now

Reply:
The function doesn't work when I take out the space after system?